frontend/src/scenes/insights/Insights.tsx

```diff
@@ -67,8 +67,11 @@ export function Insights(): JSX.Element {
     } = useValues(insightLogic)
     const { setActiveView, toggleControlsCollapsed } = useActions(insightLogic)
     const { reportHotkeyNavigation } = useActions(eventUsageLogic)
-    const { showingPeople } = useValues(trendsLogic())
-    const { refreshCohort, saveCohortWithFilters } = useActions(trendsLogic())
+    const { showingPeople } = useValues(trendsLogic({ dashboardItemId: null, view: activeView, filters: allFilters }))
```

Comment:
Think it might be cleaner to instantiate a `logic` const with the props for `trendsLogic` and just use that var for the `useValues` and `useActions` hooks (we do this in other places too)